.chloggen/elasticsearchexporter_retry-on-status-429.yaml

```
@@ -0,0 +1,28 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: elasticsearchexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Change default retry.retry_on_status to [429]
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [32584]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
+  To retain the previous behavior, set retry.retry_on_status to `[429, 500, 502, 503, 504]`.
```

This needs to be a valid Yaml, so either put the key and value on the same line, or use on of the [Yaml block style indicators](https://yaml.org/spec/1.2.2/#chapter-8-block-style-productions) like `|`.

```suggestion
subtext: To retain the previous behavior, set retry.retry_on_status to `[429, 500, 502, 503, 504]`.
```

---

Actually, looks like the changelog tool does not have any problem with that, so there's no need to fix this.